gulpfile.js

```diff
@@ -3,29 +3,19 @@ var gulp = require('gulp')
 var jshint = require('gulp-jshint')
 var babel = require('gulp-babel');
 
-gulp.task('lint', function() {
```

Comment:
what do you use to run your server? probably include it as part of your default task.